### PR TITLE
Create simple Github CI pipeline

### PR DESCRIPTION
**Pull request by p1p3**
PR to add a Github Workflow to build the project when it is PUSH to the default branch or a PR is created.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build
+
+on:
+  push:
+    branches: [ "6.0.6-ded4" ]
```
Comment (felixschuller) [inline .github/workflows/build.yml:8]: why is this limited?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
```
Comment (felixschuller) [inline .github/workflows/build.yml:29]: why not have a build/test step? why just build?

Comment (p1p3) [reply .github/workflows/build.yml:29]: Currently build does : npm run test && npm run clean:dist && npm run version && npm run build:prod && npm run build:dev

Comment (p1p3) [reply .github/workflows/build.yml:26]: Currently build does : npm run test && npm run clean:dist && npm run version && npm run build:prod && npm run build:dev

**Review by felixschuller — APPROVED** [review]
Doing gods work

**Pull request merged**